HISTORY.md

```diff
@@ -1,5 +1,22 @@
 # Release History - open AEA
 
+
+## 1.24.0 (2022-11-15)
+
+AEA:
+- Adds deprecation warning for `--aev` flag
+- Makes the usage of environment variables default
+- Extends `push-all` command to push only the development packages
+- Adds support for generating environment variable names if not provided by default
+- Updates various configuration classes to make sure path string serialization is deterministic across the various platforms 
+
+Test:
+- Fixes the tests skipped because of the wrongly configures ledger ID
+- Adds tests to make sure the configuration serialization is consistent across the various platforms
```

Comment:
I wouldn't express it this strongly. I offer no guarantee, that is to say we will need to verify hashes are in fact the same, which is currently not being tested. Furthermore we would need to test path conversion from windows-format, which is not yet done either - this can only be tested on Windows - #440